package.json

```diff
@@ -18,20 +18,13 @@
   "type": "module",
   "scripts": {
     "build": "eleventy",
-    "serve": "eleventy --serve",
-    "cypress:open": "cypress open",
-    "cypress:run": "cypress run"
+    "serve": "eleventy --serve"
   },
   "engines": {
     "node": ">=22.0.0 <23.0.0"
   },
   "dependencies": {
-    "@11ty/eleventy": "^3.1.2",
-    "@x-govuk/govuk-eleventy-plugin": "^8.0.3"
-  },
-  "devDependencies": {
-    "axe-core": "^4.10.3",
-    "cypress-axe": "^1.7.0",
-    "cypress": "^15.2.0"
+    "@11ty/eleventy": "3.1.2",
+    "@x-govuk/govuk-eleventy-plugin": "8.0.4"
```

Comment:
Updates to version tracked in other PR: https://github.com/UKHomeOffice/developer-healthcheck-docs/pull/36/files